### ISSUE TITLE
Wrong documentation related to running a custom Git Push command

### ISSUE THREAD

**Issue by lhrinaldi** [release-it/release-it#1091]
In this section it mentions that I can use `after:git:release` to execute a custom git push.

https://github.com/release-it/release-it/blob/c2230713ddaa4adce2b586a241c16e7e69fe8936/docs/git.md#further-customizations

```
{
  "git": {
    "push": false
  },
  "hooks": {
    "after:git:release": "git push origin HEAD"
  }
}
```

But then in this one https://github.com/release-it/release-it#hooks it says the `after:git:release` hook is not execute if push is disabled.

>Note that hooks like after:git:release will not run when either the git push failed, or when it is configured not to be executed (e.g. git.push: false). See [execution order](https://github.com/release-it/release-it/blob/main/docs/plugins.md#execution-order) for more details on execution order of plugin lifecycle methods.

Am I missing something here or is this wrong?

**Comment by webpro** [comment]
Do you mean that the docs are wrong or not clear, or is something not actually working as expected?

**Comment by lhrinaldi** [comment]
I think the docs are wrong, as it shows an example of using `after:git:release` to be able to use a custom push, even though this hook does not get executed when `git.push` is disabled. At least I encountered this problem when using it, but wanted to ask first before opening a PR cause I might be using it wrong.

**Comment by webpro** [comment]
Thanks for the call, @lhrinaldi, I've changed the docs.

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v17.3.0. See [Release 17.3.0](https://github.com/release-it/release-it/releases/tag/17.3.0) for release notes._
